framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -105,6 +144,12 @@ export const CreatableSelectDropdown = ({
     }
   }
 
+  useEffect(() => {
+    if (initialValue) {
+      setSelectedOption(standardOptions.find((option) => option.value === initialValue) || null)
+    }
+  }, [ initialValue ])
```

Comment:
I don't really understand what you're trying to achieve in this PR. I am pretty sure `Select` has prop `value`, which can be used to control the current state.

Name `initialValue` indicates to the developer that the value will be used only one time. And once the select component is rendered, changing the prop value from the parent component won't affect the current state.

Reply:
@olchyk98 , we need to be able to set an initial value, hence this PR and the name :) Its not required to control that value from the outside. We use CreatableSelect underneath.

Reply:
(I.e. you should have two props, one called initialValue and one just called value)